hushline/settings.py

```diff
@@ -177,9 +176,8 @@ def index() -> str | Response:  # noqa: PLR0911, PLR0912
 
             # Handle Change Password Form Submission
             if change_password_form.validate_on_submit():
-                if scrypt.verify(change_password_form.old_password.data, user.password_hash):
-                    # Hash the new password using scrypt and update the user object
-                    user.password_hash = scrypt.hash(change_password_form.new_password.data)
+                if user.check_password(change_password_form.old_password.data):
```

Comment:
A new password shouldn't be allowed if it's the same as the old password.

Reply:
Agreed, let's pick this up in a follow-on PR or open an issue for later.